### DISCUSSION TITLE
Possibiliti to get Infos from 2 different Growat / ShinePhone accounts

### DISCUSSION THREAD

**Discussion by klicky1609** [mtrossbach/noah-mqtt#48] [Ideas]
Thank's for the great work to see the Noah's in HA. 

Is there a posibility to add a secend  Growat / ShinePhone account? We have to electricity meter, two solar units and two growat accounts. It would be great to see all the data in HomeAssistent.

Marcel

**Comment by xdatravelbug** [comment]
Hi Marcel,

To give you a starter, in theory the following should work - maybe not out of the box in HA, but on other systems where you start noah-mqtt manually:
- Copy "noah-mqtt" and name it, e.g. "noah-mqtt2"
- Then start the 2nd instance with data for your 2nd account

You could try the above via command line on your HA setup => copy the 1st instance to a 2nd instance and check what happens in HA.

**Comment by les-les** [comment]
possible for a 2nd or even more noah2000  within the same growatt user account ? 

thx for helpin out

**Reply by les-les** [reply]
2 or more noah2000 within one user growatt user account works in v 0.0.28
out of the box like a charm already tested it in homeassistant
